scripts/mkcloud

```diff
@@ -644,11 +644,10 @@ listen_addr = "0.0.0.0"' >> /etc/libvirt/libvirtd.conf
 
     wait_for 300 1 "ping -q -c 1 -w 1 $adminip >/dev/null" 'crowbar admin VM'
 
-    if ! grep -q "iptables -t nat -F PREROUTING" /etc/init.d/boot.local &&
```

Comment:
OK thanks - so maybe it's enough to move the adapted code to `boot.mkcloud` and then have a flag which prevents rewriting `boot.mkcloud`?